config/environments/production.rb

```diff
@@ -3,7 +3,10 @@
   config.eager_load = true
   config.consider_all_requests_local = false
   config.action_controller.asset_host = Figaro.env.domain_name
-  config.action_controller.default_url_options = { host: Figaro.env.domain_name }
+  config.action_controller.default_url_options = {
+    locale: I18n.locale == I18n.default_locale ? nil : I18n.locale,
```

Comment:
can we make this a proc? this needs to be dynamic and evaluated with every request, not set once

Reply:
Also it's getting a code similarity error, maybe move it to `application.rb` -- that's where I try to put repeated config code across multiple environments, then override it if needed in test?